src/core/algorithms/md/hymd/preprocessing/similarity_measure/equality_similarity_measure.h

```diff
@@ -0,0 +1,104 @@
+#pragma once
+
+#include <map>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "algorithms/md/hymd/indexes/column_similarity_info.h"
+#include "algorithms/md/hymd/indexes/keyed_position_list_index.h"
+#include "algorithms/md/hymd/lowest_bound.h"
+#include "algorithms/md/hymd/preprocessing/ccv_id_pickers/pick_lhs_ccv_ids_type.h"
+#include "algorithms/md/hymd/preprocessing/similarity.h"
+#include "algorithms/md/hymd/preprocessing/similarity_measure/column_similarity_measure.h"
+#include "algorithms/md/hymd/preprocessing/similarity_measure/single_transformer.h"
```

Comment:
```suggestion
#include <cstddef>
#include <string>
#include <vector>

#include "algorithms/md/hymd/indexes/column_similarity_info.h"
#include "algorithms/md/hymd/indexes/keyed_position_list_index.h"
#include "algorithms/md/hymd/lowest_bound.h"
#include "algorithms/md/hymd/preprocessing/similarity.h"
#include "algorithms/md/hymd/preprocessing/similarity_measure/column_similarity_measure.h"
```
Move `TransformedColumnsHolder` to another header.